app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,24 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
+      if count_only
+        grouped_rates.length
+      else
+        objects = []
+        grouped_rates.sort.map do |description_group|
+          objects.push(
+            # We identified by the structure Compute_id:Storage_id
+            :id    => description_group[1][0].id.to_s + ":" + description_group[1][1].id.to_s,
```

Comment:
Sorry @h-kataria. I am having the same error, I'm doing reengineering all the code because I've made some mistakes in the process. When everything is functional again I will notify you. Thank you.